manifests/params.pp

```diff
@@ -57,8 +57,38 @@
     $functions_path     = '/etc/rc.d/init.d/functions'
     $start_pidfile_args = '--pidfile $pidfile'
   }
+  
   if $::is_pe == true or $::is_pe == 'true' {
-    # Puppet Enterprise specific settings
+    # < PE 4
+    $is_pe_server      = true
+  }elsif is_function_available('pe_compiling_server_version') {
```

Comment:
@ghoneycutt its going away as there is no such thing as PE in Puppet 4 on the agent side as its an all in one installer now